src/server_manager/web_app/ui_components/outline-share-dialog.ts

```diff
@@ -111,33 +118,31 @@ Polymer({
           inner-h-t-m-l="[[localize('share-description', 'openLink', '<a href=https://securityplanner.org/#/all-recommendations>', 'closeLink', '</a>')]]"
         ></p>
       </div>
-      <div contenteditable="" id="selectableText" style="-webkit-text-size-adjust: 100%;">
-        <p>[[localize('share-invite')]]</p>
+      <div
+        contenteditable=""
+        id="selectableInviteText"
+        style="-webkit-text-size-adjust: 100%;"
+        inner-h-t-m-l="[[localize('share-invite-html')]]"
+      ></div>
+      <div id="copyInvitationIndicator" hidden="">[[localize('share-invite-copied')]]</div>
+      <div class="button-row">
+        <paper-button id="copyButton" on-tap="copyInvite"
```

Comment:
Perhaps we should rename "invite/invitation" to "instructions"?
The invitation is not really complete without the access key.